controller/plan_controller.py

```diff
@@ -1,22 +1,17 @@
 from enum import Enum
-from typing import Iterable
+from typing import Iterable, Type
 
 from models.plan import Plan
 from models.camp import Camp
 
 
-###---- Manage Plan Menu ----
-def manage_plan_menu():
+def list_emergency_types() -> Type[Enum]:
     """
-    Vanessa
+    List all options for emergency type.
     """
     pass
 
 
-def list_emergency_types() -> Enum:
-    pass
-
-
 def create_camps(name: str) -> Camp:
     pass
```

Comment:
Feel like this changes could be reverted cuz they are irrelevant to this PR. 
Or they will be gone after rebasing?